values.yaml

```diff
@@ -122,6 +122,13 @@ configData:
     addr: :5000
     headers:
       X-Content-Type-Options: [nosniff]
+    ## https://docs.docker.com/registry/configuration/#debug
+    ## to enable metrics set debug.prometheus.enabled true
```

Comment:
These comments aren't really necessary, and could easily drift due to url changes or config path changes later on. OK to remove the above two comment lines.

Reply:
Sure, I will fix that